tests/testthat/helper-type-sum.R

```diff
@@ -1,19 +1,37 @@
-as_override_type_sum <- function(x) {
-  structure(x, class = "override_type_sum")
+as_override_type_sum_asis <- function(x) {
+  structure(x, class = "override_type_sum_asis")
 }
 
-type_sum.override_type_sum <- function(x, ...) {
+type_sum.override_type_sum_asis <- function(x, ...) {
```

Comment:
@lionel- has a way to do this with `rlang::local_bindings()` so it can be a bit more self-contained.

Reply:
Done in #287, thanks!